sdk/attestation/azure-security-attestation/test/ut/CMakeLists.txt

```diff
@@ -40,3 +42,10 @@ target_link_libraries(azure-security-attestation-test PRIVATE Azure::azure-secur
 gtest_discover_tests(azure-security-attestation-test 
     DISCOVERY_TIMEOUT 600 
     TEST_PREFIX azure-security-attestation.)
+
+
+file(COPY ${CMAKE_CURRENT_LIST_DIR}/../../../../../eng/scripts/Start-TestProxy.ps1
```

Comment:
Same as earlier - this relative path looks incredibly fragile. Is it possible to do something relative to the root directory here?